pal/pacer/pacer-web.ts

```diff
@@ -96,32 +97,41 @@ export class Pacer {
         this._stHandle = this._stTime(updateCallback);
 
         this._isPlaying = true;
+        this._frameCount = 0;
```

Comment:
What's frame count?

Reply:
_frameCount is used as a counter to determine whether a tick is needed in current frame or delayed until the next frame to tick.